doc/source/development/contributing_environment.rst

```diff
@@ -223,7 +223,10 @@ Because the meson build system is newer, you may find bugs/minor issues as it ma
 To compile pandas with meson, run::
 
    # Build and install pandas
-   python -m pip install -ve . --no-build-isolation
+   # By default, this will print verbose output
+   # showing the "rebuild" taking place on import (see section below for explanation)
+   # If you do not want to see this, omit everything after --no-build-isolation
+   python -m pip install -ve . --no-build-isolation --config-settings editable-verbose=true
```

Comment:
Somehow unrelated to the changes here, but when I `pip install -e .` I need to call Python from the same directory where I call `pip install`. So, what I do is `pip install -e $(pwd)` which adds the current package to the environment regardless of the current directory. Is people just happy to call python always from the pandas directory, or am I missing something? I feel that suggesting `$(pwd)` here in the docs could be better if the same happens to everyone.